### PR TITLE
Jenkins integration

### PR DESCRIPTION
**Pull request by donovanjamesl**

### CODE REVIEW

**Comment by donovanjamesl** [comment]
Test this please

**Comment by chrislovecnm** [comment]
Test this please

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      yaml """
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    jenkins: build-node
+spec:
+  containers:
+  - name: k8s-node
+    image: gcr.io/pso-helmsman-cicd/jenkins-k8s-node:1.0.1
```
Comment (chrislovecnm) [inline Jenkinsfile]: Need to update Jenkins container

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      steps {
+        container('k8s-node') {
+          script {
+            env.ZONE = "${ZONE}"
+            env.PROJECT_ID = "${PROJECT_ID}"
+            env.REGION = "${REGION}"
+            env.KEYFILE = GOOGLE_APPLICATION_CREDENTIALS
+          }
+          // Setup gcloud service account access
+          sh "USER=jenkins gcloud auth activate-service-account --key-file=${env.KEYFILE}"
```
Comment (chrislovecnm) [inline Jenkinsfile]: We need to not reset the environment variable.  This is a work around not a fix.

**Comment by chrislovecnm** [comment]
Test this please

**Pull request merged**